net/suricata/files/suricata.init

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh /etc/rc.common
+# (C) 2018 Ashkan Jazayeri <ashkan@jazayeri.net>
+
+START=75
+USE_PROCD=1
+
+validate_suricata_section() {
+	uci_validate_section suricata suricata "${1}" \
+		'mode:string:af-packet' \
+		'queue:list(range(0,65535))' \
+		'verbose:range(0,4):0'
+}
+
+start_service() {
+	local mode number queue verbose
+	validate_suricata_section instance || {
+		echo "validation failed"
+		return 1
+	}
+	[ -d /var/log/suricata ] || mkdir /var/log/suricata
+	[ -f /var/run/suricata.pid -a -z $(pgrep suricata) ] && rm /var/run/suricata.pid && logger -t suricata[init_script] -p daemon.alert -s "Suricata was not closed properly or it has crashed. Successfully removed the previous /var/run/suricata.pid"
```

Comment:
this seems "non-standard" at least, procd will report crashing services if they fail to start, and when it restarts them, if you're using respawn, bolting in this sort of stuff seeeeems like fluffy inconsistencies to me.